model_api/python/openvino/model_api/models/anomaly.py

```diff
@@ -0,0 +1,154 @@
+"""Definition for anomaly models.
+
+Note: This file will change when anomalib is upgraded in OTX. CVS-114640
+"""
+
+"""
+ Copyright (c) 2021-2023 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from typing import Any, Dict
+
+import cv2
+import numpy as np
+
+from .image_model import ImageModel
+from .types import ListValue, NumericalValue, StringValue
+from .utils import AnomalyResult
+
+
+class AnomalyDetection(ImageModel):
+    __model__ = "AnomalyDetection"
+
+    def __init__(self, inference_adapter, configuration=None, preload=False):
+        super().__init__(inference_adapter, configuration, preload)
+        self.output_name = self.inference_adapter.model.outputs[0].any_name
```

Comment:
self.outputs = self.inference_adapter.get_output_layers() already defined in Model, so you could access self.outputs here